sdk/keyvault/azure-security-keyvault-certificates/inc/azure/keyvault/certificates/certificate_client_operations.hpp

```diff
@@ -90,14 +90,14 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Certificat
     /**
      * @brief Cancels the operation.
      *
-     * @param context A #Azure::Core::Context controlling the request lifetime.
+     * @param context A Azure::Core::Context controlling the request lifetime.
```

Comment:
Why are you removing the anchor tag?

Reply:
Because Azure::Core::Context is in another package, and doxygen cannot figure out the URL for a type defined in another package. This can be replaced with a permalink to the docs website, but just using an anchor doesn't work :(